tests/ApiCore/Tests/Mocks/mocks.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package google.apiCore.tests.mocks;
```

Comment:
I like making this a proto!

I don't think it is normal to camelcase in the proto package option. To get the correct camelcased PHP namespace, you can use:
`php_namespace = "Google\\ApiCore\\Tests\Mocks";`